src/test/java/com/puppycrawl/tools/checkstyle/api/FilterSetTest.java

```diff
@@ -117,6 +118,16 @@ public void testUnmodifiableSet() {
             () -> filterSet.getFilters().add(filter));
     }
 
+    /*
+      This comment is not related to this code. Input based test does not call
```

Comment:
It is exactly about why this code is present.

Reply:
@romani  done.